src/plugins/condition/components/ConditionCollection.vue

```diff
@@ -35,6 +35,7 @@
             >
                 <div v-if="isEditing">
                     <ConditionEdit :condition-identifier="conditionIdentifier"
+                                   :telemetry="telemetryObjs"
                                    :current-condition-identifier="currentConditionIdentifier"
                                    @update-current-condition="updateCurrentCondition"
                                    @remove-condition="removeCondition"
```

Comment:
Let's use camelCase for events and props to be consistent with the rest of the code base. We really need to update our coding style guide.

Reply:
OK

Reply:
If I change the attribute names to camelCase I see this linting error:
`ESLint: Attribute ':currentConditionIdentifier' must be hyphenated.(vue/attribute-hyphenation)`
Do you want me to still change it?

Reply:
VueJS docs say should be kebab-case for events. Pretty sure same for props and lint objects for those: https://vuejs.org/v2/guide/components-custom-events.html#Event-Names